neurots/extract_input/input_distributions.py

```diff
@@ -85,20 +86,25 @@ def distributions(
 
     if diameter_input_morph is None:
         diameter_input_morph = filepath
-
-    if isinstance(diameter_model, str):
-        input_distributions["diameter"] = from_diameter.model(
-            load_morphologies(diameter_input_morph)
-        )
+    morphology = load_morphologies(diameter_input_morph)
+    if isinstance(diameter_model, str) and diameter_model.startswith("M"):
+        input_distributions["diameter"] = from_diameter.model(morphology)
         input_distributions["diameter"]["method"] = diameter_model
 
     elif hasattr(diameter_model, "__call__"):
-        input_distributions["diameter"] = diameter_model(load_morphologies(diameter_input_morph))
+        input_distributions["diameter"] = diameter_model(morphology)
         input_distributions["diameter"]["method"] = "external"
-    else:
+    elif (
+        isinstance(diameter_model, str) and diameter_model == "default"
+    ) or diameter_model is None:
+
         input_distributions["diameter"] = {}
         input_distributions["diameter"]["method"] = "default"
-        L.warning("No valid diameter model provided, so we will not generate a distribution.")
+        input_distributions["diameter"]["diameter"] = build_diameter_models(
+            morphology, config={"models": ["simpler"], "neurite_types": neurite_types}
```

Comment:
Is there `simple` and then `simpler` and `simplest` ? :P
I would suggest instead of simple(r) to have something that is somewhat representative of what simple signifies here.

Reply:
Ahah I had the same question :smile: 
I would also prefer a more meaningful name (I suggest `canconical-diametrizer` :stuck_out_tongue_winking_eye: ) but we would have to update `diameter-synthesis` first.

Reply:
me too! if I find the person who named that function `simpler`....